.meta/dietpi-imager

```diff
@@ -246,12 +273,134 @@ Do you want to overwrite or backup the existing file to /root/$OUTPUT_IMG_NAME.$
 			# Rename and move to /root, if source image != output image already
 			[[ /root/$OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT != $FP_SOURCE_IMG ]] && mv "$FP_SOURCE_IMG" $OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT
 
-		# Drive source
-		else
+		# Drive source, clone with dd
+		elif [[ $CLONING_TOOL == 'dd' ]]; then
 
 			G_DIETPI-NOTIFY 2 "Creating final image with actually used size: $(( $IMAGE_SIZE / 1024 / 1024 + 1 )) MiB"
 			dd if=$FP_SOURCE of=$OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT bs=1M status=progress count=$(( $IMAGE_SIZE / 1024 / 1024 + 1 ))
 
+		# Drive source, clone with Clonezilla
+		else
+
+			G_DIETPI-NOTIFY 2 'Creating final image with Clonezilla'
+			# Install required packages
+			G_AG_CHECK_INSTALL_PREREQ clonezilla partclone xorriso isolinux syslinux-common unzip curl || Exit_On_Fail
+
+			# Download last version of Clonezilla Live
+			CLONEZILLA_VERSION=$(curl -s "$CLONEZILLA_REPO/" | grep 'class="folder "' | head -n1 | cut -d\" -f2)
+			if [[ -z "$CLONEZILLA_VERSION" ]]; then
+
+				G_DIETPI-NOTIFY 1 'Could not retrieve version number of last Clonezilla Live, aborting...'
+				Exit_On_Fail
+
+			fi
+			CLONEZILLA_ZIP="clonezilla-live-${CLONEZILLA_VERSION}-amd64.zip"
+			if [[ ! -f "$CLONEZILLA_ZIP" ]]; then
+
+				G_DIETPI-NOTIFY 2 "Downloading Clonezilla Live v$CLONEZILLA_VERSION..."
+				wget "$CLONEZILLA_REPO/${CLONEZILLA_VERSION}/${CLONEZILLA_ZIP}/download" -O "$CLONEZILLA_ZIP" || Exit_On_Fail
+
+			fi
+
+			# Extract Clonezilla Live
+			unzip "$CLONEZILLA_ZIP" -d 'tmpiso' || Exit_On_Fail
+
+			# Clone disk with Clonezilla
+			if [[ ! -d tmpiso/home/partimag/$OUTPUT_IMG_NAME ]]; then
+
+				mkdir -p tmpiso/home/partimag/
+				G_DIETPI-NOTIFY 2 'Cloning disk with Clonezilla...'
+				ocs-sr -or /root/tmpiso/home/partimag --nogui -fsck -q2 -c -j2 -z1p -i 4096 -senc -sc savedisk $OUTPUT_IMG_NAME ${FP_SOURCE##*/} || Exit_On_Fail
+
+			fi
+
+			# For the sake of privacy, remove some non vital files that contain SNs and UUIDs
+			rm -f tmpiso/home/partimag/$OUTPUT_IMG_NAME/{Info*txt,*list,clonezilla-img}
+
+			# Check image
+			G_DIETPI-NOTIFY 2 'Checking Clonezilla image...'
+			ocs-chkimg -b --nogui --ocsroot /root/tmpiso/home/partimag $OUTPUT_IMG_NAME || Exit_On_Fail
+
+			# Prepare custom files used by the installer when booting in UEFI mode
+			wget "$DIETPI_REPO"/.meta/images/dietpi-background.png -O tmpiso/boot/grub/dietpi-background.png || Exit_On_Fail
```

Comment:
@sal666 
Instead of leaving the long file name on Clonezilla image, you could also shorten it there.
```
wget "$DIETPI_REPO"/.meta/images/my_goodness_is_this_long.png -O tmpiso/boot/grub/bg.png
```
My aim basically is to have meaningful file names on Git repo, so one knows what it is and what one could use it for quickly.

Reply:
OK. Done.